src/swagger-specs/policy-service.yaml

```diff
@@ -1362,53 +1507,53 @@ definitions:
         type: array
         items:
           $ref: '#/definitions/PolicyResponse'
-      marketingActionRefs:
-        description: List of marketing action URI references to which this policy applies
-        type: array
-        items:
-          type: string
-          format: uri
-      description:
-        description: >-
-          A verbal description of what this policy enforces; the reasoning behind the denial of these marketing actions in the presence of the data usage labels
-        type: string
-        example: Conditions under which we won't send marketing-based email
   CustomDatasetConstraints:
     allOf:
       - $ref: '#/definitions/DatasetConstraints'
-    properties:
-      violatedPolicies:
-        items:
-          $ref: '#/definitions/CustomPolicyResponse'
-      marketingActionRefs:
-        example: 
-          - https://platform.adobe.io:443/data/foundation/dulepolicy/marketingActions/custom/analyticsDB
+      - properties:
+          violatedPolicies:
+            items:
+              $ref: '#/definitions/CustomPolicyResponse'
+          marketingActionRef:
+            example: https://platform.adobe.io:443/data/foundation/dulepolicy/marketingActions/custom/analyticsDB
   CoreDatasetConstraintResponse:
     allOf:
       - $ref: '#/definitions/CoreMarketingActionConstraints'
       - $ref: '#/definitions/CustomDatasetConstraints'
-    properties:
-      marketingActionRefs:
-        example: 
-          - https://platform.adobe.io:443/data/foundation/dulepolicy/marketingActions/core/emailTargeting
+      - properties:
+          marketingActionRef:
+            example: https://platform.adobe.io:443/data/foundation/dulepolicy/marketingActions/core/emailTargeting
+          duleLabels:
+            example:
+              - C1
+              - C3
+              - C7
   CustomDatasetConstraintResponse:
     allOf:
       - $ref: '#/definitions/CustomMarketingActionConstraints'
       - $ref: '#/definitions/CustomDatasetConstraints'
+      - properties:
+          marketingActionRef:
+            example: https://platform.adobe.io:443/data/foundation/dulepolicy/marketingActions/custom/analyticsDB
+          duleLabels:
+            example:
+              - C1
+              - C3
+              - C7
   EntityTypeAndId:
     required:
       - entityType
       - entityId
     properties:
       entityType:
         description: >-
-          The type of entity on which to discover labels. Currently only "dataSet" is supported
+          The type of entity on which to discover labels. Currently only `dataSet` is supported.
         type: string
         enum:
           - dataSet
         example: dataSet
       entityId:
-        description: The ID of the entity on which to discover labels.
+        description: The ID of the dataset.
```

Comment:
Just to double check - is dataset the only valid entity type here? (I'd assume so, given the enum, but I just want to be 100% certain.)